### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by blerchin** [blerchin/dma-20-mfa-show#11]
Including:
* usability via screen reader
* keyboard accessibility
* distinguishable colors
* resizing text
* captions for images and videos

**Comment by lmccart** [comment]
I can help with this one

**Comment by lmccart** [comment]
I started looking at this one. Overall, we're in pretty good shape. Let me know if you have any comments, otherwise I will begin to implement. @brysonian would love your review on this list in case I missed anything.

* [x] Missing Miles alt text

* [x] I think the `<div className="title"></div>` elements should be converted to `<h1>`

* [x] Rather than using `<h3>Artist Name</h3><h2>Artwork Title</h2>`, it would make more sense to make this all one `<h2></h2>` block using `<spans>` to set off the different type treatments.

* [x] Project description components should have `<h3 class="sr-only">Project Description</h3>` blocks

* [x] Artist bio header should be `<h3>` instead of `<h4>`

* [x] Dalena page `<h4>Audio Contributions</h4>` should be `<h3>`

* [x] Dalena page mismatch between `id="audio-contributions"` and `aria-labelledby="audio-contribution"` on `<ul>` elements

* [x] Miles page `<h4>Message from Mud</h4>` should be `<h3>`

* [x] Both ArtistNav and Nav have `id="navLabel"`, need to have unique IDs. Also make text more descriptive ex: "Artist Navigation", "Site Navigation".

* [x] [Add skip to content link to bypass nav](https://a11yproject.com/posts/skip-nav-links/)

* [x] `[user-scalable="no"]` is used in the `<meta name="viewport">` element or the `[maximum-scale]` attribute is less than 5.
[Disabling zooming is problematic for users with low vision who rely on screen magnification to properly see the contents of a web page.](https://dequeuniversity.com/rules/axe/3.1/meta-viewport?application=lighthouse) cc @blerchin @hsab 

* [x] We currently have focus outlines removed for most elements. [It would be best to add these in.](https://a11yproject.com/posts/never-remove-css-outlines/) Are we ok with this design-wise? It is possible to turn on outlines only for keyboard users. I personally prefer to expose outlines because I think it brings more awareness to accessibility, but outlines for only keyboard works fine, it's up to you all. cc @dalena 

* [x] Regarding this point, @brysonian I'm wondering if the ArtistNav menu is being handled correctly, or if we should display some onscreen cue when the user is navigating through this via keyboard?

**Comment by dalena** [comment]
@lmccart regarding the focus elements, do you have an example website showcasing how it works? Is this regarding the blobs, buttons, or anything that is clickable?

**Comment by brysonian** [comment]
It certainly would be best if there was a visual indicator as well for the artist nav. And i agree on the headers!

**Comment by lmccart** [comment]
@dalena try this page: https://a11y-style-guide.com/style-guide/ press tab on your keyboard to see the focus outline move around. this is the default browser outline styling.

on this page https://p5js.org/ you can see a custom outline styling. it's generally better practice to leave the default outline rather than do custom, because it is more recognizable to users. but I think this kind of thing is acceptable still. on the p5 site, it's probably a little too far on the minimal side though.

regarding the keyboard only outlines, this means that if you're using a mouse to navigate, you don't see them. the site would look as it does not. if you use a keyboard to tab around, then they would show up.

in any case: this would not be for the blobs, only for links in the pages and site nav menu. we have an offscreen navigation menu that keyboard users will utilize to navigate to artist pages, instead of clicking on blobs.

**Comment by dalena** [comment]
@lmccart Ah I see, I think it's great to have focus elements! If we can do it for the artist nav, too then even better.
